ui/app/controllers/vault/cluster/access/oidc/keys/key.js

```diff
@@ -0,0 +1,20 @@
+import Controller from '@ember/controller';
+import { inject as service } from '@ember/service';
+import { tracked } from '@glimmer/tracking';
+
+export default class OidcKeyController extends Controller {
+  @service router;
+  @tracked isEditRoute;
```

Comment:
It feels like because `isEditRoute` is a boolean the default should be set as true or false, instead what I'm guessing is undefined? It looks like the code you have covers all the use cases so this is a more of a question than a suggestion: if tracked type is boolean should it always be defined? Thoughts?

Reply:
I think it depends on what is being tracked. It feels a little strange to me to have a default for the template view, when showing the header is entirely dependent on the route